core/spec.md

```diff
@@ -5528,6 +5549,30 @@ as defined below:
   - MUST NOT be present if the Resource's `hasdocument` model attribute is
     set to `false.
 
+##### `ancestor` Attribute
+- Type: String
+- Description: A string value that points to another version based on the
+  `versionid`-attribute.
+
+  When creating a version without explicitly setting the `ancestor`
+  attribute, the server MUST set the `ancestor` to the `versionid` of the
+  current default Version of the Resource. The `ancestor` attribute MAY also
+  be explicitly set to NULL, to indicate to the server that this Version has
```

Comment:
s/NULL/\`null\`/

Reply:
Also, we need to think more about this because today when someone sets an attribute "null" we never tell the impl to save it as "null". Meaning, a GET of that attribute later will most likely be absent in the response. If we did that for "ancestor" then on an import/put we can't tell the diff between "I'm a root" and "I come after the current default version".  We may need to require this attribute to ALWAYS be serialized to remove the ambiguity. Or, I guess, define a special versionid to mean "root" - but I'm not thrilled with that (despite that we do reserve some other version IDs - like "null" - for other edge cases).

I think the simple/default case should be "missing ancestor == I come after current default".